plugin/pkg/scheduler/scheduler_test.go

```diff
@@ -43,6 +43,8 @@ import (
 	"k8s.io/kubernetes/plugin/pkg/scheduler/volumebinder"
 )
 
+var order = []string{"VolumeBindingChecker"}
```

Comment:
This doesn't need to be file-scoped. Please put this inside the test that uses it and use `predicates.CheckVolumeBindingPred` instead of the literal string. Also, please add at least one more predicate to the list. I think `GeneralPredicates` would be a good choice.